# Hackseq Projects 2016

+ [Project 1](https://github.com/hackseq/hackseq_projects_2016/issues/10)
  Implement an evaluation framework for software that manipulates HGVS-formatted variants
+ [Project 2](https://github.com/hackseq/hackseq_projects_2016/issues/9)
  Design a tool to optimize the parameters of any command line tool
+ [Project 4](https://github.com/hackseq/hackseq_projects_2016/issues/7)
  Pseudo-WGS variant calling for common cell types aggregating ChIP-seq, RNA-seq and DHS from ENCODE and Roadmap Epigenomics data
+ [Project 5](https://github.com/hackseq/hackseq_projects_2016/issues/6)
  A framework to evaluate profiles from DNA-binding site collections represented in peak sequences from ChIP-Seq assays
+ [Project 6](https://github.com/hackseq/hackseq_projects_2016/issues/5)
  Inferring sex chromosome and autosomal ploidy in NGS data
+ [Project 7](https://github.com/hackseq/hackseq_projects_2016/issues/4)
  Visualization of single cell RNA-seq data from tens of thousands of cells in real time
+ [Project 8](https://github.com/hackseq/hackseq_projects_2016/issues/3)
  Explore the use of 10x Genomics' Linked-Reads to unlock currently inaccessible parts of the genome
+ [Project 9](https://github.com/hackseq/hackseq_projects_2016/issues/2)
  Selection of tag SNPs for an African SNP array by LD and haplotype based methods
+ [Project 10](https://github.com/hackseq/hackseq_projects_2016/issues/1)
  Develop an open-source, automated pipeline to close bacterial genomes with long read technologies
